client/prerender.config.ts

```diff
@@ -20,6 +24,6 @@ export const config: PrerenderConfig = {
     };
   },
   filterUrl(url) {
-    return !!(url && entryUrls.includes(url.pathname) && url.pathname !== "/");
+    return !(url && url.toString().includes("authenticator"));
```

Comment:
Why just the authenticator check?

Reply:
The original check wasn't really doing anything -- entryUrls is exactly the crawled URL list, and prerendering used to fail on the home page, so I'm guessing it just got left in.  Now, authenticator pages are failing because they include the Amplify authenticator component, which errors if Amplify isn't configured (and we can't configure it because of pinpoint).  I'm still looking into all of this but the simple fix for now is to just not prerender authenticator pages.